poc/gen_test_vec.py

```diff
@@ -91,42 +90,43 @@ def gen_test_vec_for_xof(cls: type[Xof]) -> None:
     from vdaf_poc import idpf_bbcggi21, vdaf_poplar1, vdaf_prio3, xof
 
     ctx = b'some application'
+    vdaf_test_vec_path = TEST_VECTOR_PATH + "/vdaf/"
```

Comment:
Just noting for posterity that this is because the repo contains test vectors for subroutines, e.g., IDPFs.